server/forms.py

```diff
@@ -240,9 +292,57 @@ class CreateTaskForm(BaseForm):
                                    default=False)
 
 class UploadSubmissionForm(BaseForm):
-    upload_files = FileField('Submission Files', [FileRequired()])
-    flag_submission = BooleanField('Flag this submission for grading',
-                                   default=False)
+    upload_files = BackupUploadField('Submission Files', [FileRequired()])
```

Comment:
Is the removal of `flag_submission` intentional? I'd be okay with it but just want to make sure. 

Students/Staff can still flag by clicking on the specific submission - but for staff specifically keeping this option might be useful if the submission they created is going to vanish in their submission history (For example staff submits something for yesterday and its now on page 2 of the history) 

^ This isn't an issue if we keep sorting by created time though.